Source/Core/ServiceTask+ObjC.swift

```diff
@@ -132,9 +132,7 @@ extension ServiceTask {
     */
     @objc public func updateUIObjC(handler: (AnyObject?) -> Void) -> Self {
         return updateUI { value in
-            if let value = value as? AnyObject {
-                handler(value)
-            }
+            handler(value as! AnyObject?)
         }
```

Comment:
Ok.  I think i misunderstood the intent.  Thanks for clearing it up for me on slack.